docs/upload.rst

```diff
@@ -333,12 +367,6 @@ filename.
 
 Records of the upload and what files were in it are available on the website.
 
-.. Note::
-
-   Symbols files can not be overwritten. Once a file is uploaded, any future
-   upload attempts for that debug_filename/debug_id combination will be
-   skipped.
```

Comment:
This was never true. This was a misunderstanding I had for a long time and should have fixed this ages ago.